src/schema/validator.js

```diff
@@ -81,6 +81,18 @@ export const validateLangPack = (...args) => {
   return isValid;
 };
 
+const _validateDictionary = validator.compile({
+  ...schemaObject,
+  id: 'dictionary-manifest',
+  $ref: '#/types/WebExtensionDictionaryManifest',
+});
+
+export const validateDictionary = (...args) => {
+  const isValid = _validateDictionary(...args);
+  validateDictionary.errors = filterErrors(_validateDictionary.errors);
+  return isValid;
+};
```

Comment:
Similarly to static themes I'd guess that dictionaries should never contain anything in the manifest from regular WebExtensions, e.g `content_script` or others should error.

We achieve this by injecting `additionalProperties: false` as you can see here: https://github.com/mozilla/addons-linter/pull/2121/files#diff-949170b1ae8ed6356b1a78fd751d87a0R111

But that can very easily be a follow-up with more validation happening for `.aff` and `.dic` files too.

Reply:
Yeah I was planning to do this in https://github.com/mozilla/addons-linter/issues/2055